src/commands/fix.ts

```diff
@@ -0,0 +1,68 @@
+import { GluegunCommand } from 'gluegun'
+import { SolidarityOutputMode, SolidarityRequirementChunk, SolidarityRunContext } from '../types'
+// Have to do this for tests rather than import
```

Comment:
I mean, I believe you, but why?

Reply:
Having just run into [something similar yesterday](https://twitter.com/jamonholmgren/status/1174832238980030464), it's probably so it can be mocked.

Reply:
Yeah I actually just copied this over from the main solidarity command so I'm not sure. I should double check that it is still true.

Reply:
That appears to not be needed anymore. 😄